build_local_windows.ps1

```diff
@@ -118,6 +118,9 @@ Set-AWSCredential -ProfileName aws_s3_windows;
 Set-DefaultAWSRegion -Region us-west-2;
 
 # Upload Tar Ball to dcos.download.io
+
 Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\$env:TEAMCITY_BRANCH\windows\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read -Metadata @{"Cache-Control" = "no-cache"};
+Write-S3Object -BucketName "downloads.dcos.io" -Key "dcos\testing\$env:TEAMCITY_BRANCH\commit\$env:BUILD_VCS_NUMBER\windows\dcos_generate_config_win.sh" -File ".\dcos_generate_config_win.sh" -CannedACLName public-read -Metadata @{"Cache-Control" = "max-age=604800"};
```

Comment:
Note the final version for Enterprise used `-HeaderCollection @{"Cache-Control" = "max-age=604800"};` rather than `-Metadata`. `-Metadata` doesn't actually set the header correctly, so that must be changed for both uploads.

Also look at the form of the Enterprise script in https://github.com/mesosphere/dcos-enterprise/blob/62d9605d5f231ac60a3bb7fce4eb26499da0379d/build_local_windows.ps1#L128-L138

The comments are useful for people to find the location of the uploaded files, and the `Write-Output` adds a link to the files that can clicked on from the TC output.

Reply:
Thanks for the comments Jon. I didn't know about this I'll set them correctly and add the comments.